src/Miunie.Json/Miunie.Json.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <CodeAnalysisRuleSet>../.ruleset</CodeAnalysisRuleSet>
```

Comment:
The ruleset reference is done globally through our `Directory.Build.props` so no need to reference it.

Reply:
You're still adding the ruleset. :sweat_smile: